docs/BuildConfiguration.md

```diff
@@ -54,6 +54,9 @@ Sections follow that describe particular settings in more depth.
 |       source                                                             |    `uri`    | The unique URI pointing to the data file.                                                                                                                      | *Required* |                                   |  2.2  |
 | demDefaults                                                              |   `object`  | Default properties for DEM extracts.                                                                                                                           | *Optional* |                                   |  2.3  |
 |    [elevationUnitMultiplier](#demDefaults_elevationUnitMultiplier)       |   `double`  | Specify a multiplier to convert elevation units from source to meters.                                                                                         | *Optional* | `1.0`                             |  2.3  |
+| [digitransitEmissions](#digitransitEmissions)                            |   `object`  | Digitransit emissions configuration.                                                                                                                           | *Optional* |                                   |   na  |
```

Comment:
Can you move these generated docs into the sandbox documentation?

Have a look at how it's done for flex: https://github.com/opentripplanner/OpenTripPlanner/blob/9b5848f595ca1f2036b9fb18961705f24b5abc53/src/test/java/org/opentripplanner/generate/doc/FlexConfigurationDocTest.java#L24

Result: https://docs.opentripplanner.org/en/dev-2.x/sandbox/Flex/